src/OctoshiftCLI.Tests/AdoApiTests.cs

```diff
@@ -37,5 +42,701 @@ public async void Get_User_Id_Invalid_Response()
             using var sut = new AdoApi(mockClient.Object);
             await Assert.ThrowsAsync<InvalidDataException>(async () => await sut.GetUserId());
         }
+
+        [Fact]
+        public async Task GetOrganizations()
+        {
+            var userId = "foo";
+            var endpoint = $"https://app.vssps.visualstudio.com/_apis/accounts?memberId={userId}?api-version=5.0-preview.1";
+            var accountsJson = "[{accountId: 'blah', AccountName: 'foo'}, {AccountName: 'foo2'}]";
+
+            var mockClient = new Mock<AdoClient>(null, null);
+
+            mockClient.Setup(x => x.GetAsync(endpoint).Result).Returns(accountsJson);
+
+            using var sut = new AdoApi(mockClient.Object);
+            var result = await sut.GetOrganizations(userId);
+
+            Assert.Equal(2, result.Count());
+            Assert.Contains(result, x => x == "foo");
+            Assert.Contains(result, x => x == "foo2");
```

Comment:
nit: Using `FluentAssertions` will make this a bit easier:
`result.Should().Contain(new[] {"foo", foo2"})`

Reply:
done